docs/application/dotnet/guides/alarm/alarms.md

```diff
@@ -56,6 +56,9 @@ To enable your application to use the alarm functionality:
 
 You can set an alarm which, when it expires, either launches an application or sends a notification to the user:
 
+> [!NOTE]
+> Since Tizen 6.0, the time period value of an alarm can be one of the values of `Tizen.Applications.AlarmStandardPeriod`. This time period value will be phase-aligned with another time period value of the alarm. For the `CreateAlarm()` method of the `Tizen.Applications.AlarmManager` class, if you use `Tizen.Applications.AlarmStandardPeriod`, the time period value of the alarm is guaranteed.
```

Comment:
@kilig 
Thank you for updating the changes.
Could you please verify  @mijin85cho's suggestion for the first part of the sentence.
For more information, see https://github.com/Samsung/tizen-docs/pull/1078#discussion_r407838712.